rewrite-python/src/main/java/org/openrewrite/python/internal/PythonPrinter.java

```diff
@@ -915,11 +915,11 @@ public J visitBlock(J.Block block, PrintOutputCapture<P> p) {
         @Override
         public J visitCase(J.Case ca, PrintOutputCapture<P> p) {
             beforeSyntax(ca, Location.CASE_PREFIX, p);
-            Expression elem = ca.getExpressions().get(0);
+            J elem = ca.getCaseLabels().get(0);
             if (!(elem instanceof J.Identifier) || !((J.Identifier) elem).getSimpleName().equals("default")) {
                 p.append("case");
             }
-            visitContainer("", ca.getPadding().getExpressions(), JContainer.Location.CASE_EXPRESSION, ",", "", p);
+            visitContainer("", ca.getPadding().getCaseLabels(), JContainer.Location.CASE_EXPRESSION, ",", "", p);
```

Comment:
The one thing I wonder about if these are populated already by the Python parser, but hard for me to say if so.

Reply:
Knut changed the Python remoting parser to use the CaseLabels back when it was introduced